yt_dlp/extractor/zaiko.py

```diff
@@ -0,0 +1,55 @@
+import json
+import html
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class ZaikoIE(InfoExtractor):
+    _VALID_URL = r'https://zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
+    _TESTS = [{
+        'url': 'https://zaiko.io/event/324868/stream/20571/20571',
+        'info_dict': {
+            'id': 324868,
+            'ext': 'mp4',
+            'title': 'ZAIKO STREAMING TEST',
+            'uploader_id': 454,
+            'timestamp': 1583809200,
+        }
+    }]
+
+    def _parse_vue_element_attr(self, name, string):
+        page_elem = self._search_regex(r'<' + name + '([^>]+)>', string, name)
+        attr = {}
+        for m in re.finditer(r'(:?[\w\-_]+)="([^"]+)"', page_elem):
+            key, value = m.groups()
+            if key[0] == ':':
+                key = key[1:]
+                value = json.loads(html.unescape(value))
+            attr[key] = value
+        return attr
```

Comment:
I like the idea here. We should use the tools already in the yt-dlp toolbox, though:
 - non-fatal lenient JSON parsing with `_parse_json()`
 - and utils imports: `extract_attributes`, `remove_start`, `unescapeHTML`
```suggestion
    def _parse_vue_element_attr(self, name, string, video_id):
        page_elem = self._search_regex(rf'(<{name}[^>]+>)', string, name)
        attrs = {}
        for key, value in extract_attributes(page_elem).items():
            attrs[remove_start(key, ':')] = self._parse_json(
                value, video_id, transform_source=unescapeHTML, fatal=False)
        return attrs
```